### PR TITLE
Linux Bulk Gateway Server

### PR DESCRIPTION
**Pull request by willwatters**

### CODE REVIEW

File: groups/live/templates/bulk_gateway_user_data.tpl [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# Redirect the user-data output to the console logs
+exec > >(tee /var/log/user-data.log|logger -t user-data -s 2>/dev/console) 2>&1
+
+#Run Ansible playbook for server setup using provided inputs
+echo '${ANSIBLE_INPUTS}' > /root/ansible_inputs.json
+/usr/local/bin/ansible-playbook /root/deployment.yml -e '@/root/ansible_inputs.json'
```
Comment (MartyFox) [inline groups/live/templates/bulk_gateway_user_data.tpl:7]: There wint be a playbook as you are using the base AMI.

File: groups/staging/templates/bulk_gateway_user_data.tpl [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# Redirect the user-data output to the console logs
+exec > >(tee /var/log/user-data.log|logger -t user-data -s 2>/dev/console) 2>&1
+
+#Run Ansible playbook for server setup using provided inputs
+echo '${ANSIBLE_INPUTS}' > /root/ansible_inputs.json
+/usr/local/bin/ansible-playbook /root/deployment.yml -e '@/root/ansible_inputs.json'
```
Comment (MartyFox) [inline groups/staging/templates/bulk_gateway_user_data.tpl:7]: Same as above.

**Review by MartyFox — APPROVED** [review]

File: groups/live/templates/bulk_gateway_user_data.tpl (hunk 1 above)
Comment (willwatters) [reply groups/live/templates/bulk_gateway_user_data.tpl:7]: The AL2 base AMI was updated to have a deployment.yml as part of the CloudWatch config.

File: groups/staging/templates/bulk_gateway_user_data.tpl (hunk 2 above)
Comment (willwatters) [reply groups/staging/templates/bulk_gateway_user_data.tpl:7]: Same as above comment.

**Pull request merged**